docs/interpreter/cassandra.md

```diff
@@ -793,7 +812,17 @@ Below are the configuration parameters and their default value.
 
 <hr/>
 
-## 14. Bugs & Contacts
+## 14. Change Log
+
+**2.0** : 
```

Comment:
The versioning of the interpreter is here to track changes/bug fixes and new features in line with Cassandra versions.

I don't see why it is confusing. Zeppelin version relates to all core/infrastructure code. Interpreter version relates to features of custom backend